src/azul/portal_service.py

```diff
@@ -41,6 +43,20 @@ def __init__(self):
         self.client = aws.client('s3')
         self.version_service = VersionService()
 
+    @property
+    def bucket(self):
+        return config.portal_db_bucket
+
+    @property
+    def object_key(self):
+        return config.portal_db_object_key(self.data_source)
+
+    @cached_property
+    def data_source(self):
+        # FIXME https://github.com/DataBiosphere/azul/issues/2716
```

Comment:
Fix formatting and add one-liner comment, potentially just the title of the issue.